File: PHP/ZMQServer.php
<?php

$context = new ZMQContext();
$socket = new ZMQSocket($context, ZMQ::SOCKET_REP);

$socket->bind("tcp://*:5555");

$pid = posix_getpid();

while(true) {
    $msg = $socket->recv();
    echo "$pid : $msg";
    $socket->send("ACK");
}

?>

File: python/ZMQServer.py
#!/usr/bin/env python2.7
import os
import zmq

context = zmq.Context()
socket = context.socket(zmq.REP)

socket.bind("tcp://*:5555")

pid = os.getpid()

while(True):
    msg= socket.recv()
    print pid, ":", msg
    socket.send("ACK")